api-src/org/labkey/api/targetedms/model/SampleFileInfo.java

```diff
@@ -0,0 +1,237 @@
+    }
+
+    public Date getAcquiredTime()
+    {
+        return acquiredTime;
+    }
+
+    public void setAcquiredTime(Date acquiredTime)
+    {
+        this.acquiredTime = acquiredTime;
+    }
+
+    public int getMetrics()
+    {
+        return metrics;
+    }
+
+    public void setMetrics(int metrics)
+    {
+        this.metrics = metrics;
+    }
+
+    public int getNonConformers()
+    {
+        return nonConformers;
+    }
+
+    public void setNonConformers(int nonConformers)
+    {
+        this.nonConformers = nonConformers;
+    }
+
+    public int getTotalCount()
+    {
+        return totalCount;
+    }
+
+    public void setTotalCount(int totalCount)
+    {
+        this.totalCount = totalCount;
+    }
+
+    public int getGuideSetId()
+    {
+        return guideSetId;
+    }
+
+    public void setGuideSetId(int guideSetId)
+    {
+        this.guideSetId = guideSetId;
+    }
+
+    public boolean isIgnoreForAllMetric()
+    {
+        return ignoreForAllMetric;
+    }
+
+    public void setIgnoreForAllMetric(boolean ignoreForAllMetric)
+    {
+        this.ignoreForAllMetric = ignoreForAllMetric;
+    }
+
+    public List<LJOutlier> getItems()
+    {
+        return items;
+    }
+
+    public void setItems(List<LJOutlier> items)
+    {
+        this.items = items;
+    }
+
+    public int getCUSUMm()
+    {
+        return CUSUMm;
+    }
+
+    public void setCUSUMm(int CUSUMm)
+    {
+        this.CUSUMm = CUSUMm;
+    }
+
+    public int getCUSUMv()
+    {
+        return CUSUMv;
+    }
+
+    public void setCUSUMv(int CUSUMv)
+    {
+        this.CUSUMv = CUSUMv;
+    }
+
+    public int getCUSUMmP()
+    {
+        return CUSUMmP;
+    }
+
+    public void setCUSUMmP(int CUSUMmP)
+    {
+        this.CUSUMmP = CUSUMmP;
+    }
+
+    public int getCUSUMvP()
+    {
+        return CUSUMvP;
+    }
+
+    public void setCUSUMvP(int CUSUMvP)
+    {
+        this.CUSUMvP = CUSUMvP;
+    }
+
+    public int getCUSUMmN()
+    {
+        return CUSUMmN;
+    }
+
+    public void setCUSUMmN(int CUSUMmN)
+    {
+        this.CUSUMmN = CUSUMmN;
+    }
+
+    public int getCUSUMvN()
+    {
+        return CUSUMvN;
+    }
+
+    public void setCUSUMvN(int CUSUMvN)
+    {
+        this.CUSUMvN = CUSUMvN;
+    }
+
+    public int getmR()
+    {
+        return mR;
+    }
+
+    public void setmR(int mR)
+    {
+        this.mR = mR;
+    }
+
+    public boolean isHasOutliers()
+    {
+        return hasOutliers;
+    }
+
+    public void setHasOutliers(boolean hasOutliers)
+    {
+        this.hasOutliers = hasOutliers;
+    }
+
+    public List<JSONObject> getItemsJSON(List<LJOutlier> ljOutliers)
+    {
+        List<JSONObject> jsonLJOutliers = new ArrayList<>();
+        for (LJOutlier ljOutlier : ljOutliers)
+        {
+            jsonLJOutliers.add(ljOutlier.toJSON());
+        }
+        return jsonLJOutliers;
+    }
+
+    public JSONObject toJSON(){
+        JSONObject jsonObject = new JSONObject();
```

Comment:
I think we should be able to use Jackson to auto-convert this to JSON

Reply:
changed and pushed.